frontend/src/components/Footer/index.jsx

```diff
@@ -67,6 +69,7 @@ export default function Footer() {
             href={paths.discord()}
             target="_blank"
             rel="noreferrer"
+            rel="noreferrer"
```

Comment:
Duplicate attributes?

Reply:
Oops missed that in the merge conflict editor 😅